src/sass/base/_statistics.scss

```diff
@@ -1,7 +1,7 @@
 figure {
   @include flex-container;
   padding: $section-margin 0;
-  margin: 0 0 $gutter;
+  margin: 100px $gutter;
```

Comment:
This will add 100px above and below the figure element. Wondering if the ticket specifications changed?